<launch>
  <arg name="topic" default="/forward/image_undistorted" />
  <arg name="username" default="tsender" />
  <arg name="file_name" default="pool_test" />
  <arg name="ext" default=".avi" /> <!--Leave as .avi-->
  <arg name="frame_rate" default="30" />
  <arg name="width" default="644" /> <!--max = 1288-->
  <arg name="height" default="482" /> <!--max = 964-->

  <node pkg="riptide_vision" type="extract_video" name="extract_video" output="screen" >
    <param name="topic" type="string" value="$(arg topic)" />
    <param name="username" type="string" value="$(arg username)" />
    <param name="file_name" type="string" value="$(arg file_name)" />
    <param name="ext" type="string" value="$(arg ext)" />
    <param name="frame_rate" type="int" value="$(arg frame_rate)" /> <!-- Required to be an int -->
    <param name="height" type="int" value="$(arg height)" />
    <param name="width" type="int" value="$(arg width)" />
  </node>
</launch>
